core/scripts/chaincli/command/feed/root.go

```diff
@@ -0,0 +1,16 @@
+package feed
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// RootCmd represents the root price feed sub-command to manage feeds
+var RootCmd = &cobra.Command{
```

Comment:
is there a reason to separate the root and subcommand? Or could we have the root 'feed' command in the same file as the subcommand 'deploy'. Seems having all the subcommands in the same file maybe easier to read.

Reply:
Good point. What if there will be more than 1 command? Or if some commands will have flags definition or some extra logic. I'm just trying to don't put everything in one place.